src/agent/rustjail/src/container.rs

```diff
@@ -1214,13 +1221,14 @@ async fn join_namespaces(
     info!(logger, "wait child received oci spec");
 
     read_async(pipe_r).await?;
+    if !p.init {
+        info!(logger, "send oci process from parent to child");
```

Comment:
@jodh-intel Regarding a unit test of these changes, I'ld like to add the test to the `integration/agent/agent_test.sh` was proposed by https://github.com/kata-containers/tests/pull/4341 because the code path of theses changes is included in `do_init_child()` function. I'll create the test after the script is merged into main.